HIP/hip-1046.md

```diff
@@ -154,11 +226,16 @@ actions is extremely low.
 
 ## How to Teach This
 
-Address Book documentation will need to be updated to reflect the new field.  
+The Address Book documentation will be updated to include the new service
+endpoint options.
+Additionally, a blog post or video will be created to explain the change,
+ensuring developers understand it and are aware that it will not affect
+existing SDK-based applications.
 
 ## Reference Implementation
 
-The reference implementation must be complete before any HIP is given the status of “Final”. The final implementation must include test code and documentation.
+The reference implementation must be complete before any HIP is given the status
+of “Final”. The final implementation must include test code and documentation.
 
 ## Rejected Ideas
```

Comment:
I would have expected a new field in `node_create.proto` and `node_update.proto` like:
```
repeated proto.ServiceEndpoint grpc_web_proxy_endpoint;
```

There are probably good reasons why this approach was not taken, even though it seems less error-prone to use. Can we please add an entry under "Rejected Ideas" explaining why this approach was not taken?